src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml

```diff
@@ -7,10 +7,9 @@ SetupMTCOpenDoorAffordance:
   end_effector_open_pose_name: "open"
 
   # List of controllers to use during the stages that interact with the door.
-  # For this objective, we want to enable admittance control.
-  door_opening_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
+  move_along_arc_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
 
-  approach_distance: 0.05
-  translation_distance: 0.0
-  rotation_distance: 60.0
+  approach_distance: 0.15   # meters
+  translation_distance: 0.15   # meters
+  rotation_distance: 50.0  # Degrees
```

Comment:
Why were these updates needed?

Also, does the translation distance addition here mean that we're pushing the door while twisting it? Very cool.

Reply:
These were present but in different configuration files. The distance has been lowered from 65 though to prevent weird pinch-planning errors for this. https://github.com/PickNikRobotics/moveit_studio/blob/main/src/moveit_studio_behavior/src/utils/affordance_transforms.cpp defines this however this portion could use refactoring when re refactor the MTC in future work.